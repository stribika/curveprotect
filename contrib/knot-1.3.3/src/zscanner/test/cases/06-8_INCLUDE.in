$ORIGIN .
$TTL	1

; KO
$INCLUDE ./includes/include6		; Include file contains include
