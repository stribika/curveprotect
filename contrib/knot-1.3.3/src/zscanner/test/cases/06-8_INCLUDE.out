OWNER=016200
CLASS=0001
RRTTL=00000E10
RTYPE=0002
RDATA=00
------
OWNER=016205746C64316100
CLASS=0001
RRTTL=00000E10
RTYPE=0002
RDATA=05746C64316100
------
